### ISSUE TITLE
16:10 cosmetic issues

### ISSUE THREAD

**Issue by mgonzales71** [marcelveldt/xbmc.skin.titan#111]
changing this issue to 16:10 cosmetic issues now that it's clear what the issue really is.

**Comment by mgonzales71** [comment]
also note: the plot does not display in the Info OSD and i checked the settings are correct to display it.

wonder it all this is just for this show - zoo?  hmmm.

![screenshot004](https://cloud.githubusercontent.com/assets/6510026/8509657/ce0b5716-2264-11e5-9773-e94f57cf401b.png)

**Comment by mgonzales71** [comment]
i think some of these issues are related to just this show for some weird reason.

other shows display info fine... strange.

**Comment by mgonzales71** [comment]
i may have to re-install helix to make sure these aren't isengard related or not but one more i have noticed...

![2015-07-05_01-00-35](https://cloud.githubusercontent.com/assets/6510026/8510825/7dd126a2-22b1-11e5-9bdc-ba32872ab25d.png)

**Comment by marcelveldt** [comment]
This is because I've added support for the 16:10 aspect ratio...
Like mentioned on the forums this could mean some alignment errors here and there..

I will fix it asap. There are some more places perhaps where the top to bottom alignment is wrong on 16:10 aspect

**Comment by mgonzales71** [comment]
oh - sorry I am not caught up with all the posts on the forum.  I didn't know that  :)  well now I do! hehe

**Comment by marcelveldt** [comment]
I just fixed the aligment issues for the OSD on git. Please look if you any others..

For your info: I've added support for 16:10 aspect ratio. The skin was originally designed for 1920x1080 so a 16:9 aspect but a lot of displays and devices nowaydays use a 16:10 AR.

Now that I've added the 16:10 AR support the skin is no longer stretched but instead uses the native resolution. This means that any controls that had a vertical alignment from the top <posy> but that should be aligned to the bottom of the screen, must be altered to use a bottom-down aligment with <bottom>

**Comment by mgonzales71** [comment]
will let you know if I see others...  it felt like something visually had changed - i need to keep up with the forums better :)

**Comment by marcelveldt** [comment]
OK thanks. Yeah the forums can be hard to follow up sometimes.

**Comment by mgonzales71** [comment]
a few more that I have noticed in the 3.3.28 rev of the skin

![2015-07-07_02-15-34](https://cloud.githubusercontent.com/assets/6510026/8543019/c4f7d780-244e-11e5-9478-7d3061b67e89.png)
![2015-07-07_02-17-06](https://cloud.githubusercontent.com/assets/6510026/8543017/c4f4ee9e-244e-11e5-8031-19c8f8bc1bb4.png)
![2015-07-07_02-12-48](https://cloud.githubusercontent.com/assets/6510026/8543018/c4f660ee-244e-11e5-8c62-1eb5e76d1577.jpg)

**Comment by marcelveldt** [comment]
all fixed in latest betas!
